
Fatal error: %s
Stack trace:
#0 %s(%d): B->go()
#1 {main}
  thrown in %s on line %d